pfio/CMakeLists.txt

```diff
@@ -112,7 +112,9 @@ else ()
   message(STATUS "netCDF does not have quantize capability")
 endif ()
 
-find_package (PFLOGGER REQUIRED)
+if (NOT BUILD_WITH_PFLOGGER)
```

Comment:
Shouldn't these be `if (BUILD_WITH_PFLOGGER)`?

Reply:
sigh. Yes. Feelin' dumb... pushed change.

Reply:
@AlexanderRichert-NOAA Please try again with, you know, the right logic 😄

Reply:
Someone is not doing a good job of taking the day off!   (But at least they are invisible on Teams!)

Reply:
Yep this looks good, thanks Matt. And if it's your day off then I better not see this updated till Tuesday :)